src/type/definition.ts

```diff
@@ -1724,14 +1701,46 @@ export interface GraphQLInputFieldConfig {
 
 export type GraphQLInputFieldConfigMap = ObjMap<GraphQLInputFieldConfig>;
 
-export interface GraphQLInputField {
+export class GraphQLInputField extends GraphQLSchemaElement {
```

Comment:
This change seems likely to break implementations that implement `GraphQLInputField` in a custom way. I don't know how common that is, but definitely not a purely safe adjustment.